benchmarks/benchmarks.py

```diff
@@ -0,0 +1,60 @@
+import betterproto
+from dataclasses import dataclass
+
+
+@dataclass
+class TestMessage(betterproto.Message):
+    foo: int = betterproto.uint32_field(0)
+    bar: str = betterproto.string_field(1)
+    baz: float = betterproto.float_field(2)
+
+class BenchMessage:
+    """
+    An example benchmark that times the performance of various kinds
+    of iterating over dictionaries in Python.
+    """
+    def setup(self):
+        self.cls = TestMessage
+        self.instance = TestMessage()
+
+    def time_overhead(self):
+        """Overhead in class definition.
+        """
+        for _ in range(100):
+            @dataclass
+            class Message(betterproto.Message):
```

Comment:
maybe nicer to factor this out into a factory function so the class def is only in the file once?

Maybe that would also allow parameterizing the benchmark with different size classes?

Reply:
Possibly, but in my experience when you try to make "fancy" things happen in benchmarks, your "fancy thing" may inadvertently affect your benchmark results.